src/sparseml/pytorch/image_classification/utils/helpers.py

```diff
@@ -116,182 +127,52 @@ def get_save_dir_and_loggers(
-            image_size=image_size,
-            **args.dataset_kwargs,
-        )
-        if args.is_main_process:
-            is_training = task == Tasks.TRAIN
-            val_loader = DataLoader(
-                val_dataset,
-                batch_size=args.test_batch_size if is_training else 1,
-                shuffle=False,
-                num_workers=args.loader_num_workers if is_training else 1,
-                pin_memory=args.loader_pin_memory if is_training else False,
-                collate_fn=_get_collate_fn(arch_key=args.arch_key, task=task),
-            )
-            if task == Tasks.EXPORT:
-                val_loader = early_stop_data_loader(
-                    val_loader, args.num_samples if args.num_samples > 1 else 1
-                )
-            print(f"created val_dataset: {val_dataset}")
-        else:
-            val_loader = None  # only val dataset needed to get the number of classes
-        return val_dataset, val_loader
-    return None, None  # val dataset not needed
-
-
-def get_train_and_validation_loaders(
-    args: Any, image_size: Tuple[int, ...], task: Optional[Tasks] = None
-):
+# data helpers
+def get_train_and_validation_loaders(args: Any, task: Optional[Tasks] = None):
     """
     :param args: Object containing relevant configuration for the task
-    :param image_size: A Tuple of integers representing the shape of input image
     :param task: The current task being performed
     :return: 4 element tuple with the following format (train_dataset,
         train_loader, val_dataset, val_loader)
     """
     train_dataset, train_loader = _create_train_dataset_and_loader(
-        args, image_size, task=task
+        args, args.image_size, task=task
     )
     val_dataset, val_loader = _create_val_dataset_and_loader(
-        args, image_size, task=task
+        args, args.image_size, task=task
     )
     return train_dataset, train_loader, val_dataset, val_loader
 
 
 # Model creation Helpers
-
-
-def get_arch_key(arch_key: Optional[str], checkpoint_path: Optional[str]) -> str:
-    """
-    Utility method to read and return the arch_key from the checkpoint, if it
-    is not passed and exists in the checkpoint. if passed the passed value is
-    returned
-
-    :param arch_key: Optional[str] The arch_key to use for the model
-    :checkpoint_path: Optiona[str] The path to the checkpoint
-    """
-    if arch_key is None:
-        if checkpoint_path:
-            checkpoint = torch.load(checkpoint_path)
-        else:
-            raise ValueError(
-                "Must provide a checkpoint path if no arch_key is provided"
-            )
-        if "arch_key" in checkpoint:
-            arch_key = checkpoint["arch_key"]
-        else:
-            raise ValueError(
-                "Checkpoint does not contain "
-                "arch_key, provide one using "
-                "--arch_key"
-            )
-
-    return arch_key
-
-
 def create_model(args: Any, num_classes: int) -> Module:
```

Comment:
```suggestion
def create_model(args: Any, num_classes: int) -> Tuple[Module, str]:
```

To be consistent with the docstring.

Reply:
Thanks! Updated

Reply:
Fixed!